test/functional/apps/visualize/_chart_types.js

```diff
@@ -46,7 +46,7 @@ export default function ({ getService, getPageObjects }) {
         'Region Map',
         'Tag Cloud',
         'Timelion',
-        'Timeseries',
+        'TSVB',
```

Comment:
Sorting, Tim, sorting..... I told you 100 times 🤣 `TSVB` before `Tag Cloud`. 
❤️

Reply:
Still learning the alphabet sorry :-( Do you have any good learning resources on that?